Makefile

```diff
@@ -39,7 +40,6 @@ RGBFIX  ?= $(RGBDS)rgbfix
 RGBGFX  ?= $(RGBDS)rgbgfx
 RGBLINK ?= $(RGBDS)rgblink
 
-
```

Comment:
Keep the two newlines. You removed these in several places of the file, but not all of them.

Reply:
42f5de1 should resolve this.